tests/Browser/Pages/StorePage.php

```diff
@@ -13,7 +13,7 @@ class HomePage extends Page
      */
     public function url()
     {
-        return '/';
+        return '/store/api-settings/';
```

Comment:
What is this `url()` needed for? And can we replace it with `route('store.home')` so that the URL path isn't defined in multiple places?

Reply:
I think it didn't work when I tried it, but I will try again :smile:

Reply:
P.S. The url is needed in the tests, when you `visit(new StorePage)`

Reply:
```suggestion
        return route('store.home');
```